src/lib/chips/chip-list.ts

```diff
@@ -76,6 +76,10 @@ export class MdChipList implements AfterContentInit, OnDestroy {
   /** Tab index for the chip list. */
   _tabIndex = 0;
 
+  /** User defined tab index.
+   * When it is not null, use user defined tab index. Otherwise use _tabIndex */
```

Comment:
Formatting on the JsDoc is slightly off; should either be one line or formatted like
```ts
  /**
   * First line of the comment and
   * the second line of the comment
   */
```